master_duff.py

```diff
@@ -9,13 +9,16 @@
 ## Otherwise, use "None" to get the aggregated stats of all players
 SHOW_OPPONENT_STATS_FOR = None
 
+## MAKE THIS SMALL TO FIND NEW STRING
 ## Configure how far back in the history of the Discord channel to fetch messages
 MSG_LIMIT = None
```

Comment:
For ease of testing on your end, you can actually probably just make this like 10 and leave `FETCH_SINCE `and `FETCH_BEFORE` as none, it will just pull the most recent 10 or *n* records